sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -266,7 +275,7 @@ export class ApplicationOfferingChangeRequestService {
         reason: true,
         applicationOfferingChangeRequestStatus: true,
         createdAt: true,
-        updatedAt: true,
+        studentActionDate: true,
```

Comment:
👍 thanks for taking care of this.

Reply:
@dheepak-aot  Thanks for suggesting the change.